dev/com.ibm.ws.app.manager/resources/com/ibm/ws/app/manager/internal/resources/AppManagerMessages.nlsprops

```diff
@@ -222,3 +222,6 @@ error.startAfter.cycle=CWWKZ0066E: The following applications cannot start becau
 error.startAfter.cycle.explanation=None of the applications can start because each one depends on one of the other applications to be started first.
 error.startAfter.cycle.useraction=Change the value of the startAfter attribute for applications involved in the cycle so that they do not have circular dependencies.
 
+dropins.app.also.configured=CWWKZ0067W: The dropins application at location {0} cannot start because it is also configured. 
```

Comment:
Change
"The dropins application at location {0} cannot start because it is also configured."
to,
"The Drop-ins application at location {0} cannot start because it is also configured."